src/bp-templates/bp-nouveau/includes/activity/ajax.php

```diff
@@ -519,8 +519,10 @@ function bp_nouveau_ajax_get_activity_objects() {
 				'user_id'      => bp_loggedin_user_id(),
 				'search_terms' => $_POST['search'],
 				'show_hidden'  => true,
-				'per_page'     => 2,
+				'per_page'     => 5,
 				'exclude'      => $exclude_groups,
+				'orderby'      => 'title',
+				'order'	       => 'ASC'
```

Comment:
I think you can use the **bp_before_groups_get_groups_parse_args** filter to make the changes and add the **orderby** and **order** parameters into the query. Maybe we can't change globally.